src/components/manager/ManagerDropBoxContent.js

```diff
@@ -143,7 +143,8 @@ const FileDisplay = ({file, tree, treeLoading}) => {
                 setLoadingFileContents(true);
                 const r = await fetch(urisByFilePath[file]);
                 if (r.ok) {
-                    setFileContents({...fileContents, [file]: await r.text()});
+                    const parsed = JSON.parse(await r.text());
```

Comment:
this doesn't work if the file isn't JSON - the viewer supports other files (READMEs, text files, etc.) too!

Reply:
good point, this is taken into account now